src/lib-components/Flexible/MediaGallery.vue

```diff
@@ -54,20 +78,32 @@ function selectItem(itemIndex: number) {
 </script>
 
 <template>
-  <!-- <section class="media-gallery"> -->
   <section
     v-if="block.mediaGallery && block.mediaGallery.length > 0"
-    class="media-gallery"
+    :class="parsedClasses"
   >
     <FlexibleMediaGalleryNewLightbox
       v-if="showLightboxModal"
       :items="block.mediaGallery"
       :selected-item="selectionIndex"
       tabindex="0"
+      :inline="inline"
       @close-modal="hideLightboxModal"
       @keydown.esc="hideLightboxModal"
     />
 
+    <section v-if="theme && fullWidthGallery" class="gallery-header">
```

Comment:
can we be more explicit here and specify theme.value = ftva? Especially if there are ftva styles that this depends on